src/Shared/FileSystem/FileSystemFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Build.Shared.FileSystem
+{
+    /// <summary>
+    /// Factory for <see cref="IFileSystemAbstraction"/>
+    /// </summary>
+    internal static class FileSystemFactory
+    {
+        /// <nodoc/>
+        public static IFileSystemAbstraction GetFileSystem()
+        {
+            // The windows-specific file system is only available on Windows Vista or higher
```

Comment:
Why is this commented out?

Reply:
Need some experimentation to prove it's worth turning on. I'll create an issue about it.